src/lib/api/StoreAPI.ts

```diff
@@ -12,34 +12,36 @@ import {
   MerchItemEdit,
   MerchItemOption,
   PlaceMerchOrderRequest,
+  RescheduleOrderPickupRequest,
 } from '@/lib/types/apiRequests';
-import type {
-  CreateCollectionPhotoResponse,
-  CreateMerchCollectionResponse,
-  CreateMerchItemOptionResponse,
-  CreateMerchItemResponse,
-  CreateMerchPhotoResponse,
-  DeleteMerchCollectionResponse,
-  DeleteMerchItemResponse,
-  EditMerchCollectionResponse,
-  EditMerchItemResponse,
-  GetAllMerchCollectionsResponse,
-  GetMerchOrdersResponse,
-  GetOneMerchCollectionResponse,
-  GetOneMerchItemResponse,
-  GetOneMerchOrderResponse,
-  GetOrderPickupEventResponse,
-  GetOrderPickupEventsResponse,
-  PlaceMerchOrderResponse,
-  PublicMerchCollection,
-  PublicMerchCollectionPhoto,
-  PublicMerchItem,
-  PublicMerchItemOption,
-  PublicMerchItemPhoto,
-  PublicMerchItemWithPurchaseLimits,
-  PublicOrder,
-  PublicOrderPickupEvent,
-  PublicOrderWithItems,
+import {
+  ApiResponse,
```

Comment:
shouldn't this also be

```suggestion
  type ApiResponse,
```